### PR TITLE
add error info

### PR DESCRIPTION
**Pull request by ostatni5**
Add the 'Show Details' button to the error state of the simulation.

### CODE REVIEW

File: src/WrapperApp/components/Simulation/SimulationStatus.tsx [hunk 1]
```diff
@@ -74,6 +74,36 @@ export default function SimulationStatus({ simulation, loadResults }: Simulation
...
+		errorWindow.document.open();
+		errorWindow.document.write(
+			`<html>
+			<head>
+				<title>Error Details</title>
+			</head>
+			<body>
+			<h1>Details</h1>			
+			<h2>Input Files</h2>
+			<h3>beam.dat</h3>
```
Comment (grzanka) [inline src/WrapperApp/components/Simulation/SimulationStatus.tsx:90]: Do we need to hardcode filenames here ?
Can't this code simply make a loop over a dictionary (alphabetically sorted with keys) and show the files ?

File: src/WrapperApp/components/Simulation/SimulationStatus.tsx [hunk 2]
```diff
@@ -74,6 +74,36 @@ export default function SimulationStatus({ simulation, loadResults }: Simulation
...
+			<pre>${simulation?.inputFiles!['beam.dat']}</pre>
+			<h3>mat.dat</h3>
+			<pre>${simulation?.inputFiles!['mat.dat']}</pre>
+			<h3>detect.dat</h3>
+			<pre>${simulation?.inputFiles!['detect.dat']}</pre>
+			<h3>geo.dat</h3>
+			<pre>${simulation?.inputFiles!['geo.dat']}</pre>
+
+			<h2>Error</h2>
+			<pre>${simulation?.shieldhitlog}</pre>
```
Comment (grzanka) [inline src/WrapperApp/components/Simulation/SimulationStatus.tsx:100]: This should go in first place when error happens. Logfile can be potentially very long (>10k lines).
Is it possible to show it in a windows with a scrollbar ?

Comment (grzanka) [reply src/WrapperApp/components/Simulation/SimulationStatus.tsx:100]: Logfiles and input files should be displayed on separate pages.

File: src/WrapperApp/components/Simulation/SimulationStatus.tsx (hunk 1 above)
Comment (ostatni5) [reply src/WrapperApp/components/Simulation/SimulationStatus.tsx:90]: Good idea

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**